### PR TITLE
Add additional info to rancher cloud cred KB

### PR DESCRIPTION
**Pull request by richard-cox**
- update to cover 2.9.3 changes
- add note regarding lack of ui notification on upgrade for existing expired tokens

@irishgordo i shamelessly stole your screenshot from https://github.com/rancher/rancher/pull/47650#issuecomment-2430691929, let me know if that's ok

### CODE REVIEW

**Comment by irishgordo** [comment]
@richard-cox  that works great :smile: :+1: - thanks for the doc addition to the kb :smile:

**Review by jillian-maroket — COMMENTED** [review]
Thank you for updating the article. Please apply the suggested wording changes.

File: kb/2024-05-17/modify_cloud_credential_ttl.md [hunk 1]
```diff
@@ -61,3 +61,13 @@ You can patch the expired Harvester cloud credentials to use a new authenticatio
   :::info important
   macOS users must use `gbase64` to ensure that the `-w` flag is supported.
   :::
+
+## Expiration of kubeconfig Tokens in Rancher 2.9.3
+
+From Rancher 2.9.3 the Rancher Management UI will warn when a Harvester cloud credential, or cluster connected to one, contains an expired token. Users will be given the option to `Renew` the token via the cloud credential / cluster menu
```
[the remark this answers is not in the document]
Comment (richard-cox) [reply kb/2024-05-17/modify_cloud_credential_ttl.md:67]: Thanks for the suggestion @jillian-maroket. The creds can be updated in two places though, so is the following amendment ok?

`In Rancher 2.9.3 and later versions, the Rancher UI displays a warning when a Harvester cloud credential or a related cluster contains an expired token. You can renew the token on the **Cloud Credentials** screen by selecting **⋮ > Renew**, or the Clusters screen by selecting **⋮ > Renew Cloud Credential**`

[the remark this answers is not in the document]
Comment (jillian-maroket) [reply kb/2024-05-17/modify_cloud_credential_ttl.md:67]: Yes, that's fine. I worked off of the screenshot and totally missed the "cluster menu" part of your draft. Thanks for catching the omission.

File: kb/2024-05-17/modify_cloud_credential_ttl.md [hunk 2]
```diff
@@ -61,3 +61,13 @@ You can patch the expired Harvester cloud credentials to use a new authenticatio
...
+
+![cc-renew](./imgs/cc-renew.png)
+
+:::note
+On upgrade existing expired Harvester cloud credentials will not contain a warning. Users can though still `Renew` their token via the resource's menu.
```
[the remark this answers is not in the document]
Comment (jillian-maroket) [reply kb/2024-05-17/modify_cloud_credential_ttl.md:72]: That works. Thanks!

**Review by jillian-maroket — COMMENTED** [review]
Both amendments LGTM. Thanks!

**Review by jillian-maroket — APPROVED** [review]

**Pull request merged**